data/brain_art.yml

```diff
@@ -0,0 +1,34 @@
+brain_art:
+  images:
+    - name: Kaleidoscope
+      filename: kaleidoscope.jpg
+      author:
```

Comment:
this file has some missing info and it'd be good to have someone from brainart SIG to fill it in

Reply:
Perfect: will cherry pick your commits, because you based them on my main branch and I could not find how to make a "clean PR" with your patch.